microservice-consul-demo/microservice-consul-demo-order/src/test/java/com/ewolff/microservice/order/logic/CustomerConsumerDrivenContractTest.java

```diff
@@ -1,26 +1,21 @@
 package com.ewolff.microservice.order.logic;
 
-import static org.junit.Assert.*;
-
-import java.util.Collection;
-
+import com.ewolff.microservice.order.OrderApp;
+import com.ewolff.microservice.order.clients.Customer;
+import com.ewolff.microservice.order.clients.CustomerClient;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.IntegrationTest;
-import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-import org.springframework.test.context.web.WebAppConfiguration;
 
-import com.ewolff.microservice.order.OrderApp;
-import com.ewolff.microservice.order.clients.Customer;
-import com.ewolff.microservice.order.clients.CustomerClient;
+import java.util.Collection;
+
+import static org.junit.Assert.*;
 
 @RunWith(SpringJUnit4ClassRunner.class)
-@SpringApplicationConfiguration(classes = OrderApp.class)
-@IntegrationTest
-@WebAppConfiguration
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT, classes = OrderApp.class)
```

Comment:
I didn't invest the time to check the backgrounds but with RANDOM_PORT the CDC tests failed. I'd suggest to open a ticket for that. Agree?